lua/kubectl/views/clusterrolebinding/definition.lua

```diff
@@ -5,7 +5,7 @@ local M = {
   ft = "k8s_clusterrolebinding",
   url = { "{{BASE}}/apis/rbac.authorization.k8s.io/v1/clusterrolebindings?pretty=false" },
   hints = {
-    { key = "<gd>", desc = "describe", long_desc = "Describe selected clusterrolebinding" },
+    { key = "<Plug>(kubectl.describe)", desc = "describe", long_desc = "Describe selected clusterrolebinding" },
```

Comment:
do we really need to list it here?
it's part of the global maps and I don't see why see it in the header

Reply:
i mean every definition that defines a global hint (such as describe)

Reply:
yeah I'll move it to the global hints window!

Reply:
also moved the g? from every ftplugin, made it work on floats or catch errors